pkg/apis/core/types.go

```diff
@@ -3733,6 +3733,8 @@ type PodStatus struct {
 	// init container will have ready = true, the most recently started container will have
 	// startTime set.
 	// More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-and-container-status
+	// A Pod can have one or more init containers, which are run before the app containers are started.
```

Comment:
Not sure if we need this but if we are going to add it, we would also need this comment [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L4632)

Reply:
Reading the issue, we should mention sidecars rather than more than one init containers. @SergeyKanzhelev